### ISSUE TITLE
Info tab not loading in Umbraco Forms

### ISSUE THREAD

**Issue by SelwynEdelbroek** [umbraco/Umbraco.Forms.Issues#1116]
In Umbraco Forms is the Info tab not loading.

Reproduction
------------

1. Install Umbraco version 13.0.0-rc2
2. Install Umbraco Forms: 13.0.0-rc2
3. Create Form
4. Open Info tab

### Bug summary

Info tab is not loading while opening an Umbraco Form. On opening the following console errors are thrown.
![image](https://github.com/umbraco/Umbraco.Forms.Issues/assets/92163864/27ed19a9-618c-45a4-8d44-bea3ee96f606)

### Specifics
See bug summary

### Steps to reproduce

1. Install Umbraco version 13.0.0-rc2
2. Install Umbraco Forms: 13.0.0-rc2
3. Create Form
4. Open Info tab

### Actual result

See bug summary

**Comment by AndyButland** [comment]
Firstly, thanks very much for testing out the RC.  However, I'm not seeing this issue I'm afraid.

Can I just to play back what I've done in case you can see any differences please?

```
dotnet new install Umbraco.Templates::13.0.0-rc2
dotnet new umbraco --name TestForms
cd .\TestForms\
dotnet add package Umbraco.Forms -v 13.0.0-rc2
dotnet run
```

Then installed using SQLite, navigated to the Forms section and clicked to create a form.  

Clicking immediately on the "Info" tab didn't show any error.

Adding a name for the form and saving, then clicking on the "Info" tab didn't show any error.

Have you upgraded from Forms RC 1 by any chance?  If so, you might need to [ensure your backoffice cache is cleared](https://www.andybutland.dev/2022/09/backoffice-cache-busting-with-umbraco.html).

**Comment by dtng95** [comment]
Hi @SelwynEdelbroek,
A similar issue has been found and the fix will be included in the next release.

**Comment by ronaldbarendse** [comment]
The same issue was reported for 12.2.0 in https://github.com/umbraco/Umbraco.Forms.Issues/issues/1128, which we'll fix in the next v12 patch and I've made sure this is already included in the final/stable 13.0.0 that just got released 👍🏻
